v1/src/test/java/com/google/cloud/teleport/spanner/ddl/DdlTest.java

```diff
@@ -788,6 +792,93 @@ public void pgTestModel() {
+        new PropertyGraph.GraphElementLabel("dummy-label-name1", propertyDeclsLabel1);
+    GraphElementTable.PropertyDefinition propertyDefinition1 =
+        new PropertyDefinition("dummy-prop-name",  "dummy-prop-name");
+    GraphElementTable.PropertyDefinition propertyDefinition2 =
+        new PropertyDefinition("aliased-prop-name",
+            "CONCAT(CAST(test_col AS STRING), \":\", \"dummy-column\")");
+    GraphElementTable.LabelToPropertyDefinitions labelToPropertyDefinitions1 =
+        new LabelToPropertyDefinitions(label1.name,
+            ImmutableList.of(propertyDefinition1, propertyDefinition2));
+
+    PropertyGraph.GraphElementLabel label2 =
+        new PropertyGraph.GraphElementLabel("dummy-label-name2", ImmutableList.of());
+    GraphElementTable.LabelToPropertyDefinitions labelToPropertyDefinitions2 =
+        new LabelToPropertyDefinitions(label2.name,  ImmutableList.of());
+
+    PropertyGraph.GraphElementLabel label3 =
+        new PropertyGraph.GraphElementLabel("dummy-label-name3", ImmutableList.of());
+    GraphElementTable.LabelToPropertyDefinitions labelToPropertyDefinitions3 =
+        new LabelToPropertyDefinitions(label3.name,  ImmutableList.of());
+
+    // Craft Node table
+    GraphElementTable.Builder testNodeTable =
+        GraphElementTable.builder()
+            .baseTableName("base-table")
+            .name("node-alias")
+            .kind(GraphElementTable.Kind.NODE)
+            .keyColumns(ImmutableList.of("primary-key"))
+            .labelToPropertyDefinitions(ImmutableList.of(labelToPropertyDefinitions1,
+                labelToPropertyDefinitions2));
+
+    // Craft Edge table
+    GraphElementTable.Builder testEdgeTable =
+        GraphElementTable.builder()
+            .baseTableName("edge-base-table")
+            .name("edge-alias")
+            .kind(GraphElementTable.Kind.EDGE)
+            .keyColumns(ImmutableList.of("edge-primary-key"))
+            .sourceNodeTable(new GraphNodeTableReference(
+                "base-table",
+                ImmutableList.of("node-key"),
+                ImmutableList.of("source-edge-key")))
+            .targetNodeTable(new GraphNodeTableReference(
+                "base-table",
+                ImmutableList.of("other-node-key"),
+                ImmutableList.of("dest-edge-key")))
+            .labelToPropertyDefinitions(ImmutableList.of(labelToPropertyDefinitions3));
+
+    // Build PropertyGraph
+    PropertyGraph.Builder propertyGraph =
+        PropertyGraph.builder()
+            .name("test-graph")
+            .addLabel(label1)
+            .addLabel(label2)
+            .addLabel(label3)
+            .addPropertyDeclaration(propertyDeclaration1)
+            .addPropertyDeclaration(propertyDeclaration2)
+            .addNodeTable(testNodeTable.autoBuild())
+            .addEdgeTable(testEdgeTable.autoBuild());
+
+    assertThat(propertyGraph.build().prettyPrint(),
+        equalToCompressingWhiteSpace("CREATE PROPERTY GRAPH test-graph "
```

Comment:
This feature is released in production spanner right?

Reply:
Yes, PropertyGraphs are currently available in PROD in Spanner:
https://cloud.google.com/spanner/docs/graph/overview